src/fusor/extract.py

```diff
@@ -3,55 +3,64 @@
 """
 
 import csv
-import logging
 from pathlib import Path
 
 from fusor.fusion_caller_models import (
     JAFFA,
     Arriba,
     Cicero,
     EnFusion,
+    FusionCaller,
     FusionCatcher,
     Genie,
     STARFusion,
 )
 
-_logger = logging.getLogger(__name__)
 
-
-def _check_if_file_exists(path: Path) -> bool:
+def _does_file_exist(path: Path) -> None:
     """Check if fusions file exists
 
     :param path: The path to the file
     :return ``True`` if the file exists, ``False`` if not
     """
     if not path.exists():
         statement = f"{path!s} does not exist"
-        _logger.error(statement)
-        return False
-    return True
+        raise ValueError(statement)
+    return
+
+
+def _process_fusion_caller_rows(
```

Comment:
I think a potential nice refactor (in the future) could be merging this and `fusion_caller_models.py`. You could have this be a method in the `FusionCaller` class

Reply:
This is a good idea. I'm going to attempt this refactor now